src/main/scala/tech/cryptonomic/conseil/generic/chain/DataTypes.scala

```diff
@@ -205,7 +205,7 @@ object DataTypes {
     orderBy: Option[List[QueryOrdering]],
     limit: Option[Int],
     output: Option[OutputType],
-    aggregation: Option[ApiAggregation] = None
+    aggregation: Option[List[ApiAggregation]]
```

Comment:
why not a `: List[ApiAggregation] = List.empty`here as in `Query`?

Reply:
because I would need to make a custom decoder for query as it endpoints generated cannot handle default values